validator/src/main/java/com/amazon/aoc/validators/AbstractStructuredLogValidator.java

```diff
@@ -113,7 +114,7 @@ protected void fetchAndValidateLogs(Instant startTime) throws Exception {
   }
 
   private void checkResult() throws BaseException {
-    if (schemasToValidate.isEmpty()) {
+    if (schemasToValidate.isEmpty() || schemasToValidate.keySet().equals(validatedSchema)) {
```

Comment:
do we need to keep the `isEmpty()` as i assume the empty keySet would equal the empty validatedSchema?

Reply:
I'm not sure how this is being used in other tests. But I do think this could be removed. There are multiple validators that are inherited from this abstract.

Reply:
okay we can try it in another PR as you suggested ✔️